Sources/MapLibreSwiftUI/Models/MapCamera/CameraState.swift

```diff
@@ -26,7 +26,7 @@ public enum CameraState: Hashable {
     case trackingUserLocationWithCourse
     
     /// Centered on a bounding box/rectangle.
-    case rect(northeast: CLLocationCoordinate2D, southwest: CLLocationCoordinate2D) // TODO: make a bounding box?
+	case rect(boundingBox: MLNCoordinateBounds, edgePadding: UIEdgeInsets)
```

Comment:
I can use `MLNCoordinateBounds` as an associated value in this enum, however I need to add Equatable & Hashable conformance. 

Can this have any side effects? Let's say in a big project these extensions are already added, wouldn't this clash and result in build errors?